### ISSUE TITLE
v8: Drawer in overlay is no longer fixed at bottom

### ISSUE THREAD

**Issue by bjarnef** [umbraco/Umbraco-CMS#5486]
After some of the latest changes the drawer in overlays is no longer fixed at bottom.

![image](https://user-images.githubusercontent.com/2919859/57967738-bf045980-7962-11e9-9643-c9b886c3bc5b.png)

**Comment by bjarnef** [comment]
It seems someone has removed `height:100%` from overlay style.

Before
```
.umb-overlay__form {
    display: flex;
    flex-wrap: nowrap;
    flex-direction: column;
    height: 100%;
}
```
![image](https://user-images.githubusercontent.com/2919859/57968749-be71c000-796e-11e9-9f05-7dbba2b06c72.png)



Now

```
.umb-overlay__form {
    display: flex;
    flex-wrap: nowrap;
    flex-direction: column;
}
```

![image](https://user-images.githubusercontent.com/2919859/57968750-bf0a5680-796e-11e9-8b79-3b36bbcffa08.png)

**Comment by nul800sebastiaan** [comment]
Fixed in #5488
